apps/meteor/app/livechat/server/api/v1/transcript.ts

```diff
@@ -18,3 +19,26 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'livechat/transcript/:rid',
```

Comment:
Any chance we can have API test for this endpoint here... please 😇

Reply:
I forgot i added this one